### DISCUSSION TITLE
Train a model for better results

### DISCUSSION THREAD

**Discussion by dimfreak4u** [Acly/krita-ai-diffusion#1141] [General]
How can I train the model with a directory full of photos to achieve more accurate results? 
I'm trying to generate content for a "Stroopwafel" (Dutch caramel cookie) brand.
Is there any guide available ?
Thank you

**Comment by nakanakapoi** [comment]
The easiest and fastest way for me to do it if to use Civitai online lora training. For $5 you can train 2 flux loras, or more for SDXL and 1.5. Downside is if you are unhappy with the result, no money back.

I personally use Kohya GUI. Many guides on Youtube for SDXL and 1.5. Flux training not yet supported in main repo. On my RTX 3060, 50 images lora training takes 8 hours. 2 days for flux.

**Reply by dimfreak4u** [reply]
Thank you

**Comment by ETCore7** [comment]
Try Pinokio Computer, it's a virtual machine where you can find and install AI stuff as found on Huggingface. In the app get Fluxgym. It's a trainer in 3 steps. You select images and assign a custom trigger word. Then the AI will describe each image, and then there is training.
